analytics_data_api/management/commands/generate_fake_course_data.py

```diff
@@ -161,15 +161,17 @@ def generate_daily_data(self, course_id, start_date, end_date):
             progress.update(1)
             date = date + datetime.timedelta(days=1)
 
-        for mode, ratio in enrollment_mode_ratios.iteritems():
+        for index, (mode, ratio) in enumerate(enrollment_mode_ratios.iteritems()):
             count = int(ratio * daily_total)
+            pass_rate = min(random.normalvariate(.45 + (.1 * index), .15), 1.0)
```

Comment:
I utilize `index` here so we can get a higher pass rate for `verified` than we have for `audit` (scales with position in `enrollment_mode_ratios); is that overly complex for this data?

I also just guessed at the baseline pass rates (from 45% for audit up to 85% for verified), LMK if there's a more accurate value we ought to use.